google-cloud/Gemfile

```diff
@@ -15,6 +15,7 @@ gem "google-cloud-dlp", path: "../google-cloud-dlp"
 gem "google-cloud-dns", path: "../google-cloud-dns"
 gem "google-cloud-env", path: "../google-cloud-env"
 gem "google-cloud-error_reporting", path: "../google-cloud-error_reporting"
+gem "google-cloud-errors", path: "../google-cloud-errors"
```

Comment:
Core should probably be listed here as well, to make sure it loads the correct version.

Reply:
Core is already listed here.